### PR TITLE
[MIG][14.0] mail_notify_bounce

### PR DESCRIPTION
**Pull request by SimoneVagile**

### CODE REVIEW

**Review by tafaRU — COMMENTED** [review]
Overall it is good but I'm wondering if this module is still needed in the face of https://github.com/odoo/odoo/issues/21119#issuecomment-559389190

**Comment by SimoneVagile** [comment]
> Overall it is good but I'm wondering if this module is still needed in the face of [odoo/odoo#21119 (comment)](https://github.com/odoo/odoo/issues/21119#issuecomment-559389190)

I think this module may be needed to notify bounced mails not only to sender, but to the contact list too.

**Review by hbrunn — APPROVED** [review]

**Review by Shide — APPROVED** [review]

**Comment by tafaRU** [comment]
@yajo could you please try again to run the merge command?

File: mail_notify_bounce/models/mail_thread.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+            _logger.info(
+                "Notifying bounce email from %s with Message-Id %s",
+                email_from,
+                message_id,
+            )
+            email = self.env["mail.mail"].create(
+                {
+                    "body_html": (
+                        u"%s<br/><br/><br/>%s<br/><br/>%s"
```
Comment (yajo) [inline mail_notify_bounce/models/mail_thread.py:63]: Change this to make the linters happy.

```suggestion
                        "%s<br/><br/><br/>%s<br/><br/>%s"
```

**Comment by tafaRU** [comment]
This PR can be closed as superseded by https://github.com/OCA/social/pull/981.

**Comment by rafaelbn** [comment]
Superseded but https://github.com/OCA/social/pull/981

**Pull request closed without merging**